Automattic/wp-calypso
# A4A > Marketplace: initial implementation of the hero section on hosting page

Closes https://github.com/Automattic/jetpack-genesis/issues/449
Closes https://github.com/Automattic/jetpack-genesis/issues/446

## Proposed Changes

This PR:

- Does initial implementation of the hero section on the new hosting page including add the new tabs for different hosts
- Creates an empty component for future developments.

Note:

- The background image animation will be implemented in a different PR.
- None of the pieces here are responsive, meaning it won't look good on all the devices. This will be fixed in a different PR.

## Why are these changes being made?

* This PR creates the initial work for the upcoming tasks for the hosting page design work

## Testing Instructions

* Open the A4A live link > Go to Marketplace - Hosting and verify:

- The UI looks as shown below
- You can switch between tabs
- The URL changes when the tab is changed

<img width="1727" alt="Screenshot 2024-07-25 at 1 43 20 PM" src="https://github.com/user-attachments/assets/8160aa22-6d85-4736-8c2a-7cef903fce2d">
<img width="1246" alt="Screenshot 2024-07-25 at 1 44 22 PM" src="https://github.com/user-attachments/assets/c5785df8-3295-401c-b7aa-14ddaeb1da7c">


## Pre-merge Checklist
